blackbox/docs/sql/reference/alter_table.txt

```diff
@@ -35,29 +36,13 @@ where ``column_constraint`` is::
                             FULLTEXT [ WITH ( analyzer = analyzer_name ) ]  }
     }
 
+
 Description
 ===========
 
-ALTER TABLE can be used to alter an existing table.
-
-``SET`` can be used to change a table parameter to a different value. Using
-``RESET`` will reset the parameter to its default value.
-
-``ADD COLUMN`` can be used to add an additional column to a table.
-
-While columns can be added at any time, adding a new
-:ref:`generated column <ref-generated-columns>` is only possible if the table
-is empty.
-
-``OPEN`` and ``CLOSE`` can be used to open or close the table, respectively.
-Closing a table prevents all operations, except ``ALTER TABLE ... OPEN``, to
-fail. Operations on closed partitions will not produce an exception, but will
-have no effect. Similarly, like ``SELECT`` and ``INSERT`` on partitioned will
-exclude closed partitions and continue working.
-
-``RENAME TO`` can be used to rename a table, while maintaining its schema and
-data. During this operation the table will be closed, and all operations upon
-the table will fail until the rename operation is completed.
+``ALTER TABLE`` can be used to modify an existing table definition. It provides
+options to alter, add or drop columns and constraints, enabling or disabling
```

Comment:
we don't support to drop columns? `OPEN` / `CLOSE` and `RENAME TO` is no longer mentioned here?

Reply:
@mfussenegger the drop columns was a careless mistake 😭 however, the `OPEN/CLOSE` and `RENAME` functionality is mentioned further down